Examples/hello-world-hummingbird-server/README.md

```diff
@@ -78,20 +76,18 @@ hello
 hello
 ```
 
-You should see log messages from the server for each request and each log line
-should include an OTel `trace_id` field.
+In the output from the OTel Collector, you should see debug messages for the received OTLP events.
 
-### Visualizing the traces using Jaeger UI
+### Viewing the log records
 
-Visit Jaeger UI in your browser at [localhost:16686](http://localhost:16686).
+The file the OTel Collector is using to export log records is mounted from the host. In another terminal window, you can
+watch the logging output using the following command:
 
-Select `example_server` from the dropdown and click `Find Traces`, or use
-[this pre-canned link](http://localhost:16686/search?service=example_server).
-
-See the traces for the recent requests and click to select a trace for a given request.
-
-Click to expand the trace, the metadata associated with the request and the
-process, and the events.
+```console
+% tail -f docker/logs/logs.json
+...
+{"resourceLogs":[{"resource":{"attributes":[{"key":"service.name","value":{"stringValue":"unknown_service"}}]},"scopeLogs":[{"scope":{"name":"swift-otel","version":"1.0.0"},"logRecords":[{"timeUnixNano":"1753295972925834000","observedTimeUnixNano":"1753295972925834000","severityNumber":9,"severityText":"info","body":{"stringValue":"Request"},"attributes":[{"key":"code.lineno","value":{"stringValue":"77"}},{"key":"code.function","value":{"stringValue":"handle(_:context:next:)"}},{"key":"hb.request.id","value":{"stringValue":"34dcb65c9fcf741e69e048cb68db875f"}},{"key":"code.filepath","value":{"stringValue":"Hummingbird/LogRequestMiddleware.swift"}},{"key":"hb.request.path","value":{"stringValue":"/hello"}},{"key":"hb.request.method","value":{"stringValue":"GET"}}],"traceId":"1bfa70b143698e35738e456d221d1863","spanId":"8707fe433a938827"}]}]}]}
```

Comment:
Having `unknown_service` here might be a bit off-putting for users reading it. Does the example not use a properly configured service name ATM? Or is the fix to include it in exports not yet complete?

Reply:
https://github.com/swift-otel/swift-otel/pull/265/commits/48b43dfe3de05e9171517c92a2fb7a0be3805d8b